index.html

```diff
@@ -14,26 +14,164 @@
+                <li><button class="lang" id="lan-1">html</button></li>
+                <li><button class="lang" id="lan-2">css</button></li>
+                <li><button class="lang" id="lan-3">javaScript</li></button></li>
+            </ul>
+
+            <button class="see" id="see-1">See Project</button>
+        </div>
+
+        <div class="works">
+            <img src="images/Snapshoot Portfolio (2).svg" alt="" class="tonic">
+            <h1 class="header-tonic">Tonic</h1>
+
+            <ul class="description">
+                <li class="canopy">CANOPY</li>
+                <li><img src="images/Counter.svg" alt="" id="counter"></li>
+                <li class="back">Back End Dev</li>
+                <li><img src="images/Counter.svg" alt="" id="counter"></li>
+                <li class="back">2015</li>
+            </ul>
+
+            <p class="more-info">A daily selection of privately 
+                personalized reads; no accounts or sign-ups required.
+            </p>
+
+            <ul class="languages">
+                <li><button class="lang" id="lan-1">html</button></li>
+                <li><button class="lang" id="lan-2">css</button></li>
+                <li><button class="lang" id="lan-3">javaScript</li></button></li>
+            </ul>
+
+            <button class="see" id="see-1">See Project</button>
+        </div>
+
+        <div class="works">
+            <img src="images/Snapshoot Portfolio (3).svg" alt="" class="tonic">
+            <h1 class="stories">Multi-Post Stories</h1>
+
+            <ul class="description">
+                <li class="canopy">CANOPY</li>
+                <li><img src="images/Counter.svg" alt="" id="counter"></li>
+                <li class="back">Back End Dev</li>
+                <li><img src="images/Counter.svg" alt="" id="counter"></li>
+                <li class="back">2015</li>
+            </ul>
+
+            <p class="more-info">A daily selection of privately 
+                personalized reads; no accounts or sign-ups required.
+            </p>
+
+            <ul class="languages">
+                <li><button class="lang" id="lan-1">html</button></li>
+                <li><button class="lang" id="lan-2">css</button></li>
+                <li><button class="lang" id="lan-3">javaScript</li></button></li>
+            </ul>
+
+            <button class="see" id="see-1">See Project</button>
+        </div>
+
+        <div class="myself">
+            <h1 class="about">About Myself</h1>
+            <p class="experience">Hello I'm a software developer! 
+                I can help you build a product , feature or website Look 
+                through some of my work and experience! If you like what you 
+                see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+
+            <p class="connect-1">LET'S CONNECT</p>
+
+            <ul class="icons-1">
+                <li><img src="images/Vector (4).svg" class="icons-2"></li>
```

Comment:
[OPTIONAL] As suggested by previous reviewer, Kindly relook at the design elements that are still overlapping. Kindly see the comparison below.

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/221238867-cbba9a84-7423-40ee-83a2-de884818ca0b.png)| ![image](https://user-images.githubusercontent.com/61361037/221238997-56402e99-e548-429a-a63b-de693301dff4.png)|